.github/workflows/build-action.yml

```diff
@@ -14,7 +14,6 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: '14'
-        cache: 'yarn'
     - run: npm install
```

Comment:
We should use `npm ci` instead of `npm install`. It will install the versions shown in `package-lock.json` (for consistency) & will be faster.